src/AsyncWebView.Uno.WinUI/AsyncWebView/AsyncWebView.cs

```diff
@@ -164,7 +164,7 @@ protected virtual void OnNavigationSucceeded(NavigationCompletedEventArgs args)
 
 		if (_logger.IsEnabled(LogLevel.Information))
 		{
-			_logger.LogInformation($"The navigation to uri '{SourceUri.AbsoluteUri}' has succeeded.");
```

Comment:
Can't we use something else when `SourceUri.AbsoluteUri` is problematic? It would be a shame to remove that info from the log message.

Reply:
The arguments that CoreWebView2NavigationCompletedEventArgs gives us doesn't have any Uri anymore. We only have access to `HttpStatusCode`, `IsSuccess`, `NavigationId` or `WebErrorStatus`. I don't know why they removed the Uri on the completedArgs.

Reply:
Okay but you weren't even using `args` for that log. You were using `SourceUri`. I'm assuming you could capture the target uri when you start the navigation and re-use it here?

Reply:
We re-designed the logs to fit with the args that we are getting right now.